google/cloud/documentai_toolbox/converters/config/converter_helpers.py

```diff
@@ -0,0 +1,485 @@
+
+def _upload_file(
+    bucket_name,
+    output_prefix,
+    file,
+) -> None:
+    r"""Uploads the converted docproto to gcs.
+
+    Args:
+        bucket_name (str):
+            Required. The bucket name.
+        output_prefix (str):
+            Required. The prefix for the location of the output folder.
+        file (str):
+            Required. The docproto file in string format.
+
+    Returns:
+        None.
+
+    """
+    storage_client = document._get_storage_client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(output_prefix)
+
+    print("Uploaded : %s\r" % output_prefix.split("/")[-1], end="")
+    blob.upload_from_string(file, content_type="application/json")
+
+
+def _get_files(blob_list, output_prefix, output_bucket, config_path: str = None):
+    download_pool = futures.ThreadPoolExecutor(10)
+    downloads = []
+    files = []
+    did_not_convert = []
+    labels = []
+    skip = ["DS_Store"]
+    prev = None
+    print("-------- Downloading Started --------")
+    for i, blob in enumerate(blob_list):
+        if "DS_Store" in blob.name:
+            continue
+        else:
+            file_path = blob.name.split("/")
+            file_path.pop()
+            doc_directory = file_path[-1]
+            file_path2 = "/".join(file_path)
+            if prev == doc_directory or f"{file_path2}/" == output_prefix:
+                continue
+
+            download = download_pool.submit(
+                _get_bytes,
+                output_bucket,
+                file_path2,
+                "annotation",
+                "config",
+                config_path,
+            )
+            downloads.append(download)
+
+            prev = doc_directory
+
+    return downloads
+
+
+def _get_docproto_files(f, project_id, location, processor_id, output_prefix):
+
+    did_not_convert = []
+    files = {}
+    unique_types = []
+    for future in f:
+        blobs = future.result()
+        docproto = convert_to_docproto_with_config(
+            annotated_bytes=blobs[0],
+            document_bytes=blobs[1],
+            schema_bytes=blobs[2],
+            project_id=project_id,
+            location=location,
+            processor_id=processor_id,
+            retry_number=1,
+            name=blobs[3],
+        )
+
+        if docproto == None:
+            did_not_convert.append(f"{output_prefix}/{blobs[3]}")
+            continue
+
+        for entity in docproto.entities:
+            if entity.type_ not in unique_types:
+                unique_types.append(entity.type_)
+
+        files[blobs[3]] = str(documentai.Document.to_json(docproto))
+
+    return files, unique_types, did_not_convert
+
+
+def _upload(files, gcs_output_path):
+    match = re.match(r"gs://(.*?)/(.*)", gcs_output_path)
+
+    if match is None:
+        raise ValueError("gcs_prefix does not match accepted format")
+
+    output_bucket, output_prefix = match.groups()
+
+    if output_prefix == None:
+        output_prefix = "/"
```

Comment:
This could be useful to add to the utilities file. There's lots of times where users have split the `gs://` path

Reply:
that might be something we can do in later refactor, i don't want to change this since it will add a lot of more changed files.